### PR TITLE
Added Docs for docker extensions via  Marketplace

### LINKED ISSUE #5709: [Docs] Add Docker Extension installation instructions via Extension Marketplace
#### Current State
The [Docker Extension docs page](https://docs.meshery.io/installation/platforms/docker-extension) describes how to use `docker` cli to install the Meshery Extension.

#### Desired State
The docs should also describe how to use the Extension Marketplace inside of Docker Desktop. Screenshot(s) should be included.

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by harkiratsm**
Signed-off-by: Harkirat Singh <multaniharry714@gmail.com>

**Description**

This PR fixes #5709

**Notes for Reviewers**

image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 0d6e8fe468f4c8936789d40b5498f9981672bb40 at: https://62bfc3d29c3edd716e7a7ace--meshery-docs-preview.netlify.app

**Comment by Rohith-Raju** [comment]
@harkiratsm Looks great !!. Just one change, could you change this to Using `Docker CLI` 😄 
image

**Comment by l5io** [comment]
🚀 Preview for commit 36317b92ed3ac0c326aef7f8caa03614fcbcd78c at: https://62c27667f83fa31eaaf231fd--meshery-docs-preview.netlify.app

**Comment by Rohith-Raju** [comment]
LGTM, good work @harkiratsm 😄

**Comment by debo19** [comment]
@harkiratsm will you share your updates on the websites call today?

**Comment by harkiratsm** [comment]
@debo19  Ok 👍

File: docs/pages/installation/docker-extension.md [hunk 1]
```diff
@@ -17,9 +17,15 @@ The Docker Extension for Meshery extends Docker Desktop’s position as the clou
...
-Navigate to the Extensions area of Docker Desktop.
+1) Navigate to the Extensions area of Docker Desktop.
 
-### Using `docker`
+2) Then navigate to Meshery in the MarketPlace and press install.
+
+<a href="{{ site.baseurl }}/assets/img/platforms/docker-extensions-marketplace.png">
+  <img style="width:350px;" src="{{ site.baseurl }}/assets/img/platforms/docker-extensions-marketplace.png">
+</a>
+
```
Comment (debo19) [inline docs/pages/installation/docker-extension.md:27]: There is one more step to this, showing Meshery in the marketplace and using the `install` button.

Comment (harkiratsm) [reply docs/pages/installation/docker-extension.md:27]: Thanks for the feedback , I would definately make those changes  ✨

**Comment by l5io** [comment]
🚀 Preview for commit 2152d0b6d96771f9a6737fc3cdcfd605ff467bd8 at: https://62c65cd677d87830e0a5e14b--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 380410b1501d13ed7303efbbca28174d3aa31279 at: https://62ca4c1c2db9047084a8490a--meshery-docs-preview.netlify.app

File: docs/pages/installation/docker-extension.md [hunk 2]
```diff
@@ -17,9 +17,25 @@ The Docker Extension for Meshery extends Docker Desktop’s position as the clou
 
 ### Using Docker Desktop
 
-Navigate to the Extensions area of Docker Desktop.
+1) Navigate to the Extensions area of Docker Desktop.
```
Comment (leecalcote) [inline docs/pages/installation/docker-extension.md:20]:
```suggestion
1) Navigate to the Extensions Marketplace of Docker Desktop.
```

File: docs/pages/installation/docker-extension.md [hunk 3]
```diff
@@ -17,9 +17,25 @@ The Docker Extension for Meshery extends Docker Desktop’s position as the clou
...
+2) From the Dashboard, select Add Extensions in the menu bar or open the Extensions Marketplace from the menu options.
+
+<a href="{{ site.baseurl }}/assets/img/platforms/docker-extension-marketplace-1.png">
+  <img style="width:350px;" src="{{ site.baseurl }}/assets/img/platforms/docker-extension-marketplace-1.png">
+</a>
+
+<a href="{{ site.baseurl }}/assets/img/platforms/docker-extension-marketplace-2.png">
+  <img style="width:350px;" src="{{ site.baseurl }}/assets/img/platforms/docker-extension-marketplace-2.png">
+</a>
+
```
Comment (leecalcote) [inline docs/pages/installation/docker-extension.md:31]:
```suggestion
```

File: docs/pages/installation/docker-extension.md [hunk 4]
```diff
@@ -17,9 +17,25 @@ The Docker Extension for Meshery extends Docker Desktop’s position as the clou
...
+3) Navigate to Meshery in the Marketplace and press install.
+
+<a href="{{ site.baseurl }}/assets/img/platforms/docker-extension.png">
+  <img style="width:550px;height:80px" src="{{ site.baseurl }}/assets/img/platforms/docker-extension.png">
```
Comment (leecalcote) [inline docs/pages/installation/docker-extension.md:35]:
```suggestion
  <img style="width:90%" src="{{ site.baseurl }}/assets/img/platforms/docker-extension.png">
```

**Comment by l5io** [comment]
🚀 Preview for commit aa04c4e554b482aff44f6e5a11bfddf2f29abd20 at: https://62ce902ddb83ad6ac6762e6c--meshery-docs-preview.netlify.app

**Review by leecalcote — CHANGES_REQUESTED** [review]
Please remove the `mesheryctl` prerequisite from this page.

**Comment by l5io** [comment]
🚀 Preview for commit 990f140f8be9b47830d1876b2e17df9ad26a5165 at: https://62d167f8a9f061183377de94--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]
Thanks, @harkiratsm 👍

**Comment by harkiratsm** [comment]
@debo19

**Review by debo19 — APPROVED** [review]
Thank you @harkiratsm :tada

**Pull request merged**